From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Zeno Albisser <zeno.albisser@digia.com>
Date: Sat, 29 Jun 2013 22:39:29 +0200
Subject: Do not modify the child path.

We currently only have a single process binary for Qt.
Therefore we should not choose a different binary based on required privileges.
However, this patch can only be a preliminary solution.
May be we should instead implement our own ChildProcessHost.
---
 content/common/child_process_host_impl.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/content/common/child_process_host_impl.cc b/content/common/child_process_host_impl.cc
index 1467536..2607496 100644
--- a/content/common/child_process_host_impl.cc
+++ b/content/common/child_process_host_impl.cc
@@ -107,7 +107,7 @@ base::FilePath ChildProcessHost::GetChildPath(int flags) {
   if (child_path.empty())
     PathService::Get(CHILD_PROCESS_EXE, &child_path);
 
-#if defined(OS_MACOSX)
+#if defined(OS_MACOSX) && 0
   DCHECK(!(flags & CHILD_NO_PIE && flags & CHILD_ALLOW_HEAP_EXECUTION));
 
   // If needed, choose an executable with special flags set that inform the
